packages/orderbook/src/config/config.ts

```diff
@@ -5,7 +5,7 @@ export const TESTNET_CHAIN_NAME = 'imtbl-zkevm-testnet';
 export const MAINNET_CHAIN_NAME = 'imtbl-zkevm-mainnet';
 
 export interface OrderbookOverrides {
-  provider?: providers.JsonRpcProvider | providers.Web3Provider;
```

Comment:
Strictly speaking yes, but overrides were only ever added to ease our development experience, they were not made for public use (although strictly they can be used).  I have confirmed the following about our two marketplace partners:
 - Token Trove use checkouts. This change is not breaking for checkouts as they do not use any overrides when initialising the orderbook from their components
 - Merit Circle do not use our SDK

So the impact here would be limited to people using the orderbook SDK for some other use case, and considering the SDK never signs or sends transactions (it can only be used to build transactions), even in the case where a user was passing in a provider, it would in no way break their experience to pass in the rpc url instead.

I have also confirmed none of our internal utils libraries use the override, except one tool that traders own for testing.